tests/unit_tests/test_wave_06.py

```diff
@@ -123,10 +123,10 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    assert item_f in tai.inventory
-    assert item_c in jesse.inventory
-    assert item_c not in tai.inventory
-    assert item_f not in jesse.inventory
+    assert [item_b, item_a, item_f] == tai.inventory
+    assert [item_e, item_d, item_c] == jesse.inventory
```

Comment:
Here again, since we expect a swap to occur and the swap order is not well-defined, we would prefer to check the list contents with `in` checks. With `in` checks, the length checks are important, but with list comparisons the length checks are redundant.